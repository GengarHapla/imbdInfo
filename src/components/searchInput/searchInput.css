.search-input__input {
    margin-right: 0.5em;
}
