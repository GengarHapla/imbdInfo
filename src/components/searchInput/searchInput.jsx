import React from "react";
import "./searchInput.css";

export default (props) => {
    return (
      <div>
        <input
          placeholder={"Search for a film"}
          className={"search-input__input"}
          onChange={props.onChange}
        />
        <button>Search</button>
      </div>
    );
}
