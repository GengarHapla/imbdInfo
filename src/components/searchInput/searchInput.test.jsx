import React from "react";
import searchInput from "./searchInput";

describe('SearchInput component', () => {
    it('renders correctly', () => {
        expect(<searchInput/>).toMatchSnapshot();
    })
})