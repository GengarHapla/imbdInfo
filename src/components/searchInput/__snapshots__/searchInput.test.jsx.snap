// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchInput component renders correctly 1`] = `<searchInput />`;
